packages/engine-core/src/binary/BinaryEngine.ts

```diff
@@ -573,10 +573,11 @@ ${chalk.dim("In case we're mistaken, please report this to us 🙏.")}`)
         const additionalFlag = this.allowTriggerPanic ? ['--debug'] : []
 
         const flags = [
+          '--datamodel',
+          this.inlineSchema,
```

Comment:
This may fail with [`E2BIG`](https://man7.org/linux/man-pages/man3/errno.3.html) with large schemas, writing to a temporary file might be a better option (at least if the schema size is more than a few kilobytes).

Reply:
Indeed